src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/CryptoHelpers.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace System.Security.Cryptography.Xml
+{
+    internal static class CryptoHelpers
+    {
+        public static object CreateFromName(string name)
+        {
+            switch (name)
```

Comment:
@anthonylangsworth I believe this will be here mostly as a workaround for now. I've just created an issue to remove this class. We should keep changes to minimum but in some cases we can add thin wrappers like this. I think the term "structural changes" was referring to redundant changes like refactoring or moving stuff around for no particular reasons. Let's think about additional features like additional transforms as a next step after we get everything working and properly tested - maybe a joint issue for all stuff like that. If something is necessary to get us to a working place it is ok to add it but we should make it visible (i.e. I wouldn't put it somewhere in the middle of big PR like this but a separate PR instead with link to a particular issue).